google/api/control/caches.py

```diff
@@ -210,9 +211,9 @@ def __setitem__(self, key, value, **kw):
     def out_deque(self):
         """The :class:`collections.deque` to which expired items are added."""
         self.expire()
-        expired = dict((k, v) for (k, v) in self._tracking.items()
+        expired = dict((k, v) for (k, v) in list(self._tracking.items())
```

Comment:
This line can be reverted. We want an iterator here anyway, so items() (the py2 equivalent of iteritems) is fine here.